packages/help-center/src/components/history-recorder.tsx

```diff
@@ -0,0 +1,11 @@
+import { useDispatch } from '@wordpress/data';
+import { useHistory } from 'react-router-dom';
+import { HELP_CENTER_STORE } from '../stores';
+
+export function HistoryRecorder() {
+	const history = useHistory();
+	const { setRouterState } = useDispatch( HELP_CENTER_STORE );
+	setRouterState( history.entries, history.index );
+
+	return null;
+}
```

Comment:
This component records every navigation in the help-center store, the help-center store then persist that.